modules/data-streams/src/yamlRestTest/resources/rest-api-spec/test/data_stream/190_failure_store_redirection.yml

```diff
@@ -32,13 +44,6 @@ teardown:
 
 ---
 "Redirect ingest failure in data stream to failure store":
-  - requires:
```

Comment:
Is this not required any longer? or is this just a part of the muting process?

Reply:
I moved it to the setup section because it applies the whole suite. If we need to update it it's so much easier. If later we need to make it more specific per test case, we could change this.